src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -138,13 +152,15 @@ export default Vue.extend<{}, {}, Computed, Props>({
                     const key = [current.areaId, ...this.nonAreaFilters.map(f => current.filterValues[f.id])].join("_");
                     if (!(key in displayRows)) {
                         const areaLabel =  this.flattenedAreas[current.areaId].label;
+                        const areaHierarchy = findPath(current.areaId, this.countryAreaFilterOption).replace(/undefined\//g, "")
```

Comment:
These `undefined`s are coming in because on this line: https://github.com/mrc-ide/hint/pull/297/files#diff-607cf6b7d106639467b8b36fabc00bfcR131 `obj.label` is not defined where `obj` is an array. I think replacing 

```
if (path) return obj.label + "/" + path;
```

with 

```
if (path) {
      return (obj.label ? obj.label + "/": "") + path;   
}
```

should give you want you need and then you can remove this `replace`. Side note - as a matter of convention we always use brackets for if/else statements, even if they are 1 liners. Keeps things readable and less error prone.